pkg/reader/price_reader.go

```diff
@@ -104,24 +114,28 @@ func (pr *OnchainTokenPricesReader) GetFeeQuoterTokenUpdates(
 	return updateMap, nil
 }
 
-func (pr *OnchainTokenPricesReader) GetTokenFeedPricesUSD(
+func (pr *priceReader) GetFeedPricesUSD(
 	ctx context.Context, tokens []ocr2types.Account,
 ) ([]*big.Int, error) {
 	prices := make([]*big.Int, len(tokens))
+	if pr.feedChainReader == nil {
+		pr.lggr.Debugf("node does not support feed chain")
```

Comment:
Nit: Log feed chain selector here

Reply:
We already log this on a higher level. We shouldn't even reach this point. Just extra safety.